src/rules/selector-no-qualifying-type/README.md

```diff
@@ -0,0 +1,89 @@
+# selector-no-qualifying-type
+
+Disallow qualifying a selector by type.
+
+```css
+    a.class {}
+/** ↑
+ * This is the type in a selector that is qualified with a class */
```

Comment:
Should say `"This type selector is qualifying the class"`, I think, because the rule is targeting the qualifying type.